### PR TITLE
Mainly style fixes

### PR DESCRIPTION
**Pull request by anderseknert**

### CODE REVIEW

File: lib/strings.rego [hunk 1]
```diff
@@ -14,26 +14,19 @@ ascii_letters := concat("", [ascii_lowercase, ascii_uppercase])
...
+capitalize(word) = sprintf("%v%v", [
+	upper(substring(word, 0, 1)),
+	substring(word, 1, count(word) - 1),
+])
 
 # Repeated string s x number of times
-repeat(str, x) = r {
-	r := concat("", [p | numbers.range(0, x - 1)[_]; p := str])
-}
+repeat(str, x) = concat("", [p | numbers.range(0, x - 1)[_]; p := str])
```
Comment (srenatus) [inline lib/strings.rego]: [nit]
```suggestion
repeat(str, x) = concat("", [str | numbers.range(0, x - 1)[_]])
```

File: lib/lists.rego [hunk 2]
```diff
@@ -4,11 +4,7 @@ package lists
 append(l, x) = array.concat(l, [x])
 
 # Returns a new list with item x inserted at position p of list l
-insert(l, x, p) = r {
-	first := array.slice(l, 0, p)
-	last := array.slice(l, p, count(l))
-	r := array.concat(array.concat(first, [x]), last)
-}
+insert(l, x, p) = array.concat(array.concat(array.slice(l, 0, p), [x]), array.slice(l, p, count(l)))
```
Comment (srenatus) [inline lib/lists.rego:7]: I was playing with this a bit and came up with another variant 😄 
```rego
insert_(l, x, p) = [at_index_or_list(i, p, x, l) | numbers.range(0, count(l)+2)[i]]

at_index_or_list(i, idx, _, list) = list[i] {
  i < idx
}
at_index_or_list(i, idx, _, list) = list[i-1] {
  i > idx
}
at_index_or_list(i, i, item, _) = item
```

Just sharing this, I don't think it's better. Merely less `array.*` 😀

**Review by srenatus — APPROVED** [review]
Yay

File: lib/strings.rego (hunk 1 above)
Comment (anderseknert) [reply lib/strings.rego]: Good one!

File: lib/lists.rego (hunk 2 above)
Comment (anderseknert) [reply lib/lists.rego:7]: Fun stuff!

**Pull request merged**